CMakeLists.txt

```diff
@@ -7,22 +7,55 @@ cmake_minimum_required(VERSION 3.6)
 # Note: CMake is mainly meant for usage under Windows, while under Unix we
 # prefer to use autotools; hence assume for Unix this file has quirks.
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 file(
      GLOB
      sources
      "src/libmeasurement_kit/*/*.c"
      "src/libmeasurement_kit/*/*.cpp"
     )
 
+add_definitions(-DMK_CA_BUNDLE="mk-ca-bundle.pem")
+
 if(APPLE)
   set(openssl_include_path /usr/local/Cellar/openssl/1.0.2h_1/include)
+  set(osx_extra_libs event_pthreads event_openssl)
+  set(osx_link_path /usr/local/Cellar/openssl/1.0.2h_1/lib)
```

Comment:
The above is quite *ghetto*, but the point of this file is to enable compilation on Windows and building on macOS for now is just something I do to make sure I haven't broken anything.